app/src/main/java/com/github/se/polyfit/ui/screen/PostInfoScreen.kt

```diff
@@ -74,15 +70,15 @@ fun PostInfoScreenContent(
       }
       return@Scaffold
     }
-    if (posts.isEmpty()) {
+    if (posts.value.isEmpty()) {
       NoPost()
       return@Scaffold
     }
 
     LazyColumn(
         state = rememberLazyListState(index),
     ) {
-      items(posts) { post -> PostCard(post = post) }
+      items(posts.value.size) { index -> PostCard(post = posts.value[index]) }
```

Comment:
i think you can avoid having to use the index to make clearer
```suggestion
      items(posts.value) { post -> PostCard(post = post) }
```